File: c3-platform/src/org/aphreet/c3/platform/storage/StorageException.scala
package org.aphreet.c3.platform.storage

class StorageException(val message:String, val cause:Throwable) extends RuntimeException(message, cause){

  def this(message:String) = this(message, null)
  
  def this() = this(null, null)
  
  def this(cause:Throwable) = this(null, cause)
  
}

File: c3-web/src/org/aphreet/c3/web/exception/StorageException.java
package org.aphreet.c3.web.exception;

public class StorageException extends RuntimeException{

	private static final long serialVersionUID = -6859298488394069956L;

	public StorageException(){
		super();
	}
	
	public StorageException(String msg){
		super(msg);
	}
	
	public StorageException(Throwable t){
		super(t);
	}
	
	public StorageException(String msg, Throwable t){
		super(msg, t);
	}
}
